src/content/wiki/buildings/smeltery.mdoc

```diff
@@ -5,7 +5,14 @@ building: smeltery
 
 The Smeltery is where the Smelter will smelt ores into ingots.
 
-**Hint:** The higher the level of the Smeltery, the more furnaces the Smelter will be able to use. A higher level will also have a higher chance to double and even triple the ingot output per block of ore. In 1.16, the Smelter's Strength level will sometimes determine the chance to double or triple the ingot output of ores, according to the Smeltery level. In 1.18+, the smelter applies Fortune to the ores they smelt instead. See the Smelter's page for how their strength affects the output.
+**Hint:** The higher the level of the Smeltery, the more furnaces the Smelter will be able to use.  
```

Comment:
This hint will probably have to be moved over into the version block of 1.18.2 right now. And the 1.19 version block will have to mention something different. Something like `The skill level of the Smelter will determine how many furnaces the Smelter will be able to use, up to a limit of 10.`